README.md

```diff
@@ -45,6 +45,14 @@ See deployment steps in the [Deployment Guide](DEPLOYMENT.md).
 |-----------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------|
 | [SageMaker JumpStart Foundation Model Endpoint Module](modules/fmops/sagemaker-jumpstart-fm-endpoint/README.md) | Creates an endpoint for a SageMaker JumpStart Foundation Model. |
 
+
+### MWAA Modules
+
+| Type                                                                    | Description                                                                                                                                                                                       |
+|-------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|  [Amazon Managed Workflows for Apache Airflow (MWAA) Module](https://github.com/awslabs/idf-modules/blob/main/modules/orchestration/mwaa/README.md)  |  Deploys an Amazon MWAA module   |
```

Comment:
Nitpick: can we please remove this line and reference the IDF MWAA module in the description or the example DAG module below, just adding a note that it is using MWAA module from IDF?